### PR TITLE
Update Twilio module for Ballerina Swan Lake

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
Update Twilio module for Ballerina Swan Lake

### CODE REVIEW

File: src/twilio/twilio_constants.bal [hunk 1]
```diff
@@ -65,4 +55,4 @@ final string AUTH_TOKEN = "AUTH_TOKEN";
 final string AUTHY_API_KEY = "AUTHY_API_KEY";
 
 //Error codes
-final string TWILIO_ERROR_CODE = "(ballerinax/twilio)TwilioError";
+public const TWILIO_ERROR = "(ballerinax/twilio)TwilioError";
```
Comment (chamil321) [inline src/twilio/twilio_constants.bal:58]: Do we need this error code anymore?

Comment (ldclakmal) [reply src/twilio/twilio_constants.bal]: Fixed with https://github.com/ballerina-platform/module-ballerinax-twilio/pull/64/commits/c894bd88aad5f91ac76beb5d3b8a29dac5c67007

File: src/twilio/twilio_utils.bal [hunk 2]
```diff
@@ -84,3 +81,14 @@ function convertToBoolean(json|error value) returns boolean {
     }
     return false;
 }
+
+function prepareError(string message, error? err = ()) returns Error {
+    log:printError(message, err);
```
Comment (Bhashinee) [inline src/twilio/twilio_utils.bal:86]: Do we need to log the error here?

Comment (ldclakmal) [reply src/twilio/twilio_utils.bal]: Yes. That's not needed. Removed with https://github.com/ballerina-platform/module-ballerinax-twilio/pull/64/commits/c84729156582e6ec0d1b310fa6ccf11f105b9a6b

**Pull request merged**